rentals.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date, :book
```

Comment:
- [ ] Kindly make sure `Getters` and `Setters` for `@person` instance variables are present i.e you should update the the  `attr_accessor` by adding the `:person` attribute